web-ui/src/components/member_selector/member_selector_dialog/MemberSelectorDialog.jsx

```diff
@@ -374,7 +403,7 @@ const MemberSelectorDialog = ({ open, selectedMembers, onClose, onSubmit }) => {
           </IconButton>
           <div className="toolbar-title-container">
             <Typography className="toolbar-title" variant="h5">
-              Select Members
+              Select {memberDescriptor ? memberDescriptor : 'Members'}
```

Comment:
This could probably be tightened up with `||` though I tend to prefer the ternary. Happy to adjust.